Makefile

```diff
@@ -1,4 +1,4 @@
 install: ${GOBIN}/makex
 
-${GOBIN}/makex: $(shell find -type f -and -name '*.go')
+${GOBIN}/makex: $(shell /usr/bin/find -type f -and -name '*.go')
```

Comment:
Yes, on Windows in 99% of cases `\Windows\System32\find.exe` will be used (performs text search). I doubt that somebody wants to use `find` from `/usr/local` or other non-standard script/binary from the `PATH` and I think that `/usr/bin` is the correct location everywhere